src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/RemoveUnusedLocalsTests.Fixer.cs

```diff
@@ -30,5 +32,484 @@ protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
             return new CSharpRemoveUnusedLocalsFixer();
         }
+    
+        [Fact]
+        public void UnusedLocal_BaseFunctionality_CSharp()
+        {
+            var code = @"
+class C
+{
+    int M()
+    {
+        int a = 0; // remove also comment
+        int b = 0;
+        b = 0;
```

Comment:
Will the local `b` be marked as unused if right hand side of this assignment was a method call? If so, do we ensure that when we remove that assignment statement, the method call is still retained (as it can have side effects)?

Reply:
Yes, sure. 'b' is used if there is a method call. This is covered by _Parameters_ tests. Renamed them into _ParamtersAndMethodCalls_

---
In reply to: [152381284](https://github.com/dotnet/roslyn-analyzers/pull/1336#discussion_r152381284) [](ancestors = 152381284)